Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/transform/impl/EventGraphMapper.java

```diff
@@ -291,6 +305,10 @@ protected Node copyGraph(Node start) {
 			((Relation) copiedNode).setPredicate(copyGraph(predicate));
 			((Relation) copiedNode).setObject(copyGraph(object));
 		}
+		else {
+		    copiedNode.setAlternativeIds(start.getAlternativeIds());
```

Comment:
Since alternativeIds is a property of node, you can just put that in line 198. no need for an else.